src/modules/menu/components/timeslider/TimeSlider.vue

```diff
@@ -267,13 +267,13 @@ export default {
         this.setCurrentYearAndDispatchToStore(initialYear)
     },
     beforeUnmount() {
-        this.clearPreviewYear()
+        this.clearPreviewYear({ dispatcher: 'TimeSlider.vue' })
```

Comment:
a const for this one would make sense as it is used 3 times

Reply:
Done